watchdog-config/extra-sql-queries.yml

```diff
@@ -75,3 +75,59 @@ users_with_insificient_disk:
         usage: "LABEL"
     - cht_instance:
         usage: "LABEL"
+
+days_since_successful_purge:
+  query: |
+    SELECT 
+      purge.partner_name,
+      replace(monitoring_urls.url, 'https://', '') as cht_instance,
+      now()::date - MAX(completion_date) FILTER(WHERE error IS NULL)::date AS "days"
+    FROM
+      public.app_monitoring_purge_logs as purge
+    LEFT JOIN
+      monitoring_urls ON purge.partner_name=monitoring_urls.partner_name
+    GROUP BY
+      purge.partner_name,
+      cht_instance
+  metrics:
+    - days:
+        usage: "COUNTER"
+    - partner_name:
+        usage: "LABEL"
+    - cht_instance:
+        usage: "LABEL"
+
+form_completions_60_days:
```

Comment:
Also, this is not a big deal, but it is frustrating me a bit that I cannot come up with a way to do this...

Instead of ending up with separate `incomplete_count` and `complete_count` metrics here, I think the more proper (from a Prometheus best practices perspective) would be to just include both values in the same metric, but distinguish them by a `status` label or something.  For example, our existing `cht_messaging_outgoing_total` metric contains values for all messages, but they are partitioned via a status label into `delivered/due/failed/muted...`.  Unfortunately, I cannot figure out how to recreate similar behavior with this SQL config.  Not a big deal, though!

Reply:
also fair!  but also resolving w/ no fix...